doc/kernel/usermode/usermode.rst

```diff
@@ -9,6 +9,142 @@ are defined, and how memory may be managed to support user mode threads.
 For details on creating threads that run in user mode, please see
 :ref:`lifecycle_v2`.
 
+Threat Model
+============
+
+For threads that have either started initially, or dropped down to user mode,
```

Comment:
Maybe best to just say:

    For threads in user mode, we aim to protect against the following: